config/smartcosmos-ext-interactions.yml

```diff
@@ -0,0 +1,14 @@
+server:
+  port: ${smartcosmos.service.ext-interactions.port}
+
+smartcosmos:
+  security:
+    enabled: true
+  endpoints:
+    interactions:
+      enabled: true
+      create.enabled: true
+  interactions:
```

Comment:
If the above settings are default (they should), remove them from this settings file.